src/sentry/integrations/middleware/hybrid_cloud/parser.py

```diff
@@ -53,22 +52,16 @@ def __init__(
         self.error = error
 
 
-class BaseRequestParser(abc.ABC):
```

Comment:
no `@abstractmethod`s so `ABC` doesn't make sense (and just adds runtime overhead)